Controllers/TaskController.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Controllers;
+
+use Models\TaskModel;
+
+class TaskController implements IController
+{
+    private $model;
+    private $fc;
+
+    public function __construct()
+    {
+        $this->model = new TaskModel();
+        $this->fc = FrontController::getInstance();
```

Comment:
4. Don't call frontcontroller inside another controllers. If u need to get some params from there make it workable another way